hercules-ci-agent/pkg.nix

```diff
@@ -1,15 +1,63 @@
-{ mkDerivation, aeson, async, attoparsec, base, base64-bytestring
-, bdw-gc, binary, binary-conduit, bytestring, cachix, cachix-api
-, conduit, conduit-extra, containers, directory, exceptions
-, filepath, hercules-ci-api, hostname, hspec, http-client
-, http-client-tls, http-conduit, inline-c, inline-c-cpp, katip
-, lens, lens-aeson, lifted-async, lifted-base, monad-control, mtl
-, network, nix-derivation, nix-expr, nix-main, nix-store
-, optparse-applicative, process, protolude, safe-exceptions
-, servant, servant-auth-client, servant-client, servant-client-core
-, stdenv, stm, system-filepath, temporary, text, time, tomland
-, transformers, transformers-base, unix, unliftio, unliftio-core
-, unordered-containers, uuid
+{ mkDerivation
+, aeson
+, async
+, attoparsec
+, base
+, base64-bytestring
+, bdw-gc
+, binary
+, binary-conduit
+, bytestring
+, cachix
+, cachix-api
+, conduit
+, conduit-extra
+, containers
+, directory
+, exceptions
+, filepath
+, hercules-ci-api
+, hostname
+, hspec
+, http-client
+, http-client-tls
+, http-conduit
+, inline-c
+, inline-c-cpp
+, katip
+, lens
+, lens-aeson
+, lifted-async
+, lifted-base
+, monad-control
+, mtl
+, network
+, nix-derivation
+, nix-expr
+, nix-main
+, nix-store
+, optparse-applicative
+, process
+, protolude
+, safe-exceptions
+, servant
+, servant-auth-client
+, servant-client
+, servant-client-core
+, stdenv
+, stm
+, system-filepath
+, temporary
+, text
+, time
+, tomland
+, transformers
+, transformers-base
+, unix
+, unliftio
+, unliftio-core
+, unordered-containers
+, uuid
```

Comment:
Not sure if desirable. Perhaps `generate-nix` should call the formatter.

Reply:
This will be gone with haskell.nix, so let's ignore formatting for now to avoid conflicts

Reply:
I've let this be, it's going to be easier to just delete these files once we get there.

For now the workflow is the same as now, you write changes and run pre-commit to format.